react/features/chat/components/native/Chat.tsx

```diff
@@ -17,6 +19,21 @@ import styles from './styles';
 
 interface IProps extends AbstractProps {
 
+    /**
+     * The number of unread file messages.
+     */
+    _nbUnreadFiles: number;
```

Comment:
Is "nb" supposed to mean "number" here? What ever happened to shortening number like "num"? NB (Capitalized) means Nota Bene, aka, "note well".

Reply:
Can you pl rename ther others too? 🙏

Reply:
wow 12 files changed on that matter, will open a separate PR